charts/posthog/templates/clickhouse_instance.yaml

```diff
@@ -72,9 +72,17 @@ spec:
     podTemplates:
       - name: pod-template-with-volumes
         spec:
+          securityContext:
+            runAsUser: 101
```

Comment:
What's 101, where did that value come from?

Reply:
https://github.com/PostHog/charts-clickhouse/issues/33 - it's the new default from clickhouse docker